addons/web/static/src/core/model_selector/model_selector.scss

```
@@ -0,0 +1,10 @@
+.o_sp_input_dropdown {
+    .o-autocomplete--dropdown-menu {
+        width: 25ch;
+        max-height: 27% !important;
+        .o-autocomplete--dropdown-item a {
+            text-overflow: ellipsis;
+            max-width: 85%;
```

```suggestion
            width: inherit;
```
might be more correct imho.